src/main/resources/fxml/Slette.fxml

```diff
@@ -1,16 +1,54 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import javafx.geometry.*?>
+<?import java.lang.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
 <?import javafx.scene.control.Button?>
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.layout.AnchorPane?>
-<AnchorPane xmlns:fx="http://javafx.com/fxml/1" prefHeight="200.0" prefWidth="350.0"
-            style="-fx-background-color: #FFE0B2;" xmlns="http://javafx.com/javafx/8">
+
+<AnchorPane prefHeight="230.0" prefWidth="353.0" style="-fx-background-color: #FFE0B2;" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
     <children>
-        <Button fx:id="NeiKnapp" layoutX="200.0" layoutY="135.0" mnemonicParsing="false" prefHeight="26.0"
-                prefWidth="102.0" text="Nei" onAction="#handleNoAction"/>
-        <Label layoutX="50.0" layoutY="84.0" text="Er du sikker på at du vil slette &lt;emne&gt;?"/>
-        <Button fx:id="JaKnapp" layoutX="50.0" layoutY="135.0" mnemonicParsing="false" prefHeight="26.0"
-                prefWidth="102.0" text="Ja" onAction="#handleYesAction"/>
-        <Label layoutX="124.0" layoutY="27.0" prefHeight="26.0" prefWidth="102.0" text="SLETTE AVTALE"/>
+      <GridPane stylesheets="@css/default.css" prefHeight="230.0" prefWidth="353.0" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
+        <columnConstraints>
+          <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
+          <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
+        </columnConstraints>
+        <rowConstraints>
+          <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
+          <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
+            <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
```

Comment:
Her bør det helst være lik indent hele veien. Fire mellomrom (standard tabs i intellij) gir best lesbarhet og er standard i resten av prosjektet. Du kan sette indents for hele prosjektet med høyreklikk -> reformat code.